web/src/components/Carousel.jsx

```diff
@@ -0,0 +1,65 @@
+import { h, Component } from 'preact';
+import Flickity from 'flickity';
+import 'flickity/css/flickity.css';
+
+export default class Carousel extends Component {
+  constructor(props) {
+    super(props);
+
+    this.carousel = null;
+    this.flkty = null;
+  }
+
+  create() {
+    if (this.carousel) {
+      this.flkty = new Flickity(this.carousel, this.props.options);
+
+      if (this.props.flickityRef) {
```

Comment:
prop name here is confusing.`Ref` implies this is a component reference (eg `useRef` or `createRef`), but this is actually a callback function.